src/main/java/executor/command/CommandDelete.java

```diff
@@ -7,9 +7,11 @@
 public class CommandDelete extends Command {
     protected String userInput;
 
+
     // Constructor
     public CommandDelete(String userInput) {
         this.userInput = userInput;
+        this.description = "Deletes the specific entry that the user wants to remove";
```

Comment:
Descriptions should ideally include the format which the user should input the command so they can immediately use it. Command Delete uses an index-based deletion so it should be something like Delete <index_of_task_to_delete>.

// Sample
Deletes a specific entry based on the index provided. (E.g "Delete 2" will delete the 2nd task on your task list)